KIPs/kip-fungible_token.md

```diff
@@ -0,0 +1,203 @@
+
+```solidity
+function transfer(address _to, uint256 _value) public returns (bool success)
+```
+
+#### transferFrom
+
+The `transferFrom` method is used for a withdraw workflow, allowing contracts to transfer tokens on your behalf.
+This can be used for example to allow a contract to transfer tokens on your behalf and/or to charge fees in sub-currencies.
+The function SHOULD `throw` unless the `_from` account has deliberately authorized the sender of the message via some mechanism.
+
+*Note* Transfers of 0 values MUST be treated as normal transfers and fire the `Transfer` event.
+
+```solidity
+function transferFrom(address _from, address _to, uint256 _value) public returns (bool success)
+```
+
+
+#### approve
+
+Allows `_spender` to withdraw from your account multiple times, up to the `_value` amount. If this function is called again, it overwrites the current allowance with `_value`.
+
+**NOTE**: To prevent attack vectors like the one [described here](https://docs.google.com/document/d/1YLPtQxZu1UAvO9cZ1O2RPXBbT0mooh4DYKjA_jp-RLM/) and discussed [here](https://github.com/ethereum/EIPs/issues/20#issuecomment-263524729),
+clients SHOULD make sure to create user interfaces in such a way that they set the allowance first to `0` before setting it to another value for the same spender.
+THOUGH The contract itself shouldn't enforce it, to allow backwards compatibility with contracts deployed before.
+
+```solidity
+function approve(address _spender, uint256 _value) public returns (bool success)
+```
+
+
+#### allowance
+
+Returns the amount which `_spender` is still allowed to withdraw from `_owner`.
+
+```solidity
+function allowance(address _owner, address _spender) public view returns (uint256 remaining)
+```
+
+### Events
+#### Transfer
+
+MUST trigger when tokens are transferred including zero value transfers.
+
+A token contract which mints new tokens Must trigger a Transfer event with the `_from` address set to `0x0` when tokens are created.
+A token contract which burns tokens Must trigger a Transfer event with the `_to` address set to `0x0` when tokens are burned.
+
+```solidity
+event Transfer(address indexed _from, address indexed _to, uint256 _value)
+```
+
+#### Approval
+
+MUST trigger on any successful call to `approve(address _spender, uint256 _value)`.
+
+```solidity
+event Approval(address indexed _owner, address indexed _spender, uint256 _value)
+```
+
+
+
+## Rationale
+<!--The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+This token standard is ERC-20 compliant. If you want to use ERC-20 for Klaytn, you can use this token standard.
```

Comment:
It would be better to specify differences between this token standard and ERC-20 here. What are the differences?

Reply:
I mentioned the differences in `## Specification` section like below.
```
Comparing with ERC-20 token standard, this token standard enforces the triggering of transfer events when minting / burning tokens. 
```

Reply:
Yeah, I saw it, but it's little bit vague. When I see the sentence, it pops up in my mind "what other differences? Is this the only differences between ERC-20 and this?"

Reply:
I added mint/burn function and explain the differences like below.
```
Comparing with ERC-20 token standard, this token standard includes mint/burn functions and enforces the triggering of transfer events when minting / burning tokens.
```